packages/headless-ui/vaul/test/src/app/controlled/page.tsx

```
@@ -0,0 +1,206 @@
+                  >
+                    this tweet.
+                  </a>
+                </p>
+              </div>
+            </div>
+            <div className="p-4 bg-zinc-100 border-t border-zinc-200 mt-auto">
+              <div className="flex gap-6 justify-end max-w-md mx-auto">
+                <a
+                  className="text-xs text-zinc-600 flex items-center gap-0.25"
+                  href="https://github.com/emilkowalski/vaul"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)